### PR TITLE
fix(range): screen reader cursor highlights entire range element

### PR DESCRIPTION
**Pull request by sean-perkins**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
  - Some docs updates need to be made in the `ionic-docs` repo, in a separate PR. See the [contributing guide](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#modifying-documentation) for details.
- [x] Build (`npm run build`) was run locally and any changes were pushed
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

With VoiceOver enabled, the voice over cursor will highlight the range knob instead of the entire range element when focused. 

Issue URL: Internal

## What is the new behavior?

- With VoiceOver enabled, the voice over cursor will highlight the entire range element when focused.

|Main|PR|
|----|----|
|<video src="https://user-images.githubusercontent.com/13732623/195183596-36b607b9-3738-4c99-8863-cabda045ad2f.mp4"></video>|<video src="https://user-images.githubusercontent.com/13732623/195183557-f52abadf-1444-422d-9b32-d2ce8c95260d.mp4"></video>|

Unsure why the range bis thicker between the recordings. Styling was not updated between branches, so may be the result of caching between branch checkouts. 

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: core/src/components/range/range.tsx [hunk 1]
```diff
@@ -613,8 +621,10 @@ export class Range implements ComponentInterface {
             />
           ))}
 
-          <div class="range-bar" role="presentation" part="bar" />
-          <div class="range-bar range-bar-active" role="presentation" style={barStyle} part="bar-active" />
+          <div class="range-bar-container">
```
Comment (sean-perkins) [inline core/src/components/range/range.tsx:624]: I need to remove this, this was for testing purposes.

**Review by liamdebeasi — COMMENTED** [review]
This changes the swipe behavior:

| `main` | branch |
| - | - |
| <video src="https://user-images.githubusercontent.com/2721089/195653573-bae05805-9e50-40ab-b4cb-894ea88e5e4e.mp4"></video> | <video src="https://user-images.githubusercontent.com/2721089/195653604-85770c36-8bc6-492f-9bc6-ff3b3c822ba3.mp4"></video> |

Notice that the pin seems to jump to wherever your finger is when swiping is activated.

**Comment by sean-perkins** [comment]
@amandaejohnston the desired outcome is to closer align with the native experience. This was reported from an enterprise customer when comparing against the iOS settings (FW-1894).

---

I will need to look in further detail how moving where the aria attributes are reflected, is impacting the start value when activating swipe.

**Comment by sean-perkins** [comment]
Closing this PR based on our recent discussions of a 7.x feature that may help resolve this.

**Pull request closed without merging**